src/Internal/IntType.elm

```diff
@@ -0,0 +1,11 @@
+module Internal.IntType exposing (Config)
+
+
+type alias Config intType =
```

Comment:
I think we need a better name than `IntType`, any suggestions?

Reply:
I decided for `IntType` over `IntLike`, would you like `IntLike` better? We could also drop this module altogether and inline the type in the Internal.Int32 and Internal.Int64 modules. I'm open for suggestions.
Or do you mean the type variable name `intType`? Same applies - I don't have any better names right now, but I'm open to suggestions.

Reply:
I was thinking `IntOps` or `IntOperations` would make sense here. And I would use lowercase `int` for the int type (rather than `intType`) throughout the codebase. The module name could be `Internal.Types` and I would put `Int64` here as well.

Reply:
I decided for IntOperations and named the module Internal.IntOperations since we wanted to seperate the operations from the data types. I also use `int` as the type variable name consistently now.